api/v1/commonaudit_types.go

```diff
@@ -41,12 +41,14 @@ type CommonAuditSpecFluentd struct {
 	ImageRegistry string                      `json:"imageRegistry,omitempty"`
 	PullPolicy    string                      `json:"pullPolicy,omitempty"`
 	Resources     corev1.ResourceRequirements `json:"resources,omitempty"`
+	ZenEnabled    bool                        `json:"zenEnabled,omitempty"`
```

Comment:
Is a ZenEnabled variable required ... or is it just a temporary until we get the certificates/secrets worked out?  I am expecting the Common Audit Logging will be configured/behave the same regardless of whether Zen Is installed/deployed or not.

Reply:
@ajstreit I have added it temporarily for testing purposes until we get clarified on zen certificates/secrets. we will remove it in the final push